projects/github-actions/test-results-to-slack/README.md

```diff
@@ -0,0 +1,27 @@
+# Test results to Slack GitHub Action
+
+This GitHub Action will send Slack messages with the tests results.
+
```

Comment:
You'll probably want to extend that readme a bit :)

Reply:
I left it like this on purpose, to avoid editing it too often. The plan is to add documentation when the project is done.